components/o-buttons/stories/button-group.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import {withDesign} from 'storybook-addon-designs';
+import {Button} from '../src/tsx/button';
+import {ButtonGroup} from '../src/tsx/group';
+import './button.scss';
+import withHtml from 'origami-storybook-addon-html';
+import { nanoid } from 'nanoid'
+
+export default {
+	title: 'Components/o-buttons',
+	component: ButtonGroup,
+	decorators: [withDesign, withHtml],
+	args: {},
+	parameters: {
+		design: {},
+		guidelines: {},
+		html: {},
+	},
+};
+
+const ButtonGroupStory = args => <ButtonGroup>
+	{args.buttons.map(buttonProps => <Button {...buttonProps} key={nanoid()}/>)}
```

Comment:
Are we able to remove this usage of nanoid? I believe the key value will be different on every render which is not what is expected, the key is meant to be consistent between renders

Reply:
That's right. I'm struggling for an appropriate key here though, label+type would probably do but isn't unique really

Reply:
The buttonProps object is unique with regards to each button and is also consistent between renders, let's use that?

```suggestion
	{args.buttons.map(buttonProps => <Button {...buttonProps} key={buttonProps}/>)}
```

Reply:
Ooops I didn't push the last commit. Using the props is what I've done but as a string. It's not guaranteed to be unique because it's possible to add a button with the same props using the stories controls. It'll do for our purpose, but doesn't seem ideal

Reply:
The `buttonProps` object looks to be the ideal as it is different for each button and is consistent between renders. Even if each button has the same label and type, their object would be different